usecases/objects/get.go

```diff
@@ -90,9 +94,13 @@ func (m *Manager) GetObjectsClass(ctx context.Context, principal *models.Princip
 	return s.GetClass(schema.ClassName(res.ClassName)), nil
 }
 
-func (m *Manager) getObjectFromRepo(ctx context.Context, id strfmt.UUID,
-	additional additional.Properties) (*search.Result, error) {
-	res, err := m.vectorRepo.ObjectByID(ctx, id, search.SelectProperties{}, additional)
+func (m *Manager) getObjectFromRepo(ctx context.Context, class string, id strfmt.UUID,
+	additional additional.Properties) (res *search.Result, err error) {
```

Comment:
I would avoid named return values here in this method

Reply:
https://dave.cheney.net/practical-go/presentations/gophercon-israel.html#_avoid_named_return_values

Reply:
I see your point but It is a common go practice